tests/PagesAreFoundTest.php

```diff
@@ -2,21 +2,53 @@
 
 namespace App\Tests;
 
+//use App\Repository\UserRepository;
+use Symfony\Component\Panther\Client;
 use Symfony\Component\Panther\PantherTestCase;
 
 class PagesAreFoundTest extends PantherTestCase
 {
-    private const ROUTES = [
-        '/',
-    ];
+    private const LOGGED_IN_EMAIL = 'accounts@sumocoders.be';
 
-    public function testPagesAreFound()
+    /**
+     * @dataProvider providePublicUrls
+     */
+    public function testPublicPagesAreFound($route): void
     {
         $client = static::createPantherClient();
+        $client->request('GET', $route);
+        self::assertEquals(200, $client->getInternalResponse()->getStatusCode());
+    }
+
+    /**
+     * @dataProvider provideLoggedInUrls
+     */
+    public function testLoggedInPagesAreFound($route): void
+    {
+        $client = static::createPantherClient();
+        $this->loggin($client);
+        $client->request('GET', $route);
+        self::assertEquals(200, $client->getInternalResponse()->getStatusCode());
+    }
+
+    public function providePublicUrls()
+    {
+        return [
+            ['/']
+        ];
+    }
+
+    public function provideLoggedInUrls()
+    {
+        return [
+            ['/']
+        ];
+    }
 
-        foreach ($this::ROUTES as $route) {
-            $client->request('GET', $route);
-            self::assertEquals(200, $client->getInternalResponse()->getStatusCode());
-        }
+    private function loggin(Client $client): void
```

Comment:
Wat is dit?

Reply:
Is sowieso "login"

Reply:
@tijsverkoyen is en typo moet login zijn. Dit is een method die men kan aanspreken om pagina's die achter een login zit te kunnen testen zonder door heel de login flow te gaan